transport/httptransport/auth.go

```diff
@@ -0,0 +1,21 @@
+package httptransport
+
+import "encoding/base64"
+
+func BasicAuthHeader(username, password string) string {
+	return "Basic " + BasicAuth(username, password)
+}
+
+//
+// Copied from
```

Comment:
Why do we need to copy this ? Can't we depend on it ?

Reply:
It's a private function in the http library (lowercase first letter)